### ISSUE TITLE
Beeminder--Habitica To-Dos integration failing

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#5965]
email from Daniel Reeves at Beeminder:

Subject: to-dos not getting picked up by beeminder

users are hollering today about their to-dos not getting picked up by beeminder.
quick questions as we debug:
1. any known issues on your end today?
2. confirm https://habitica.com/api/v2/ is the right base url for api calls?

**Comment by Alys** [comment]
There are no issues that I know of.
That is the correct URL.

I'm setting up my own goal for testing. If there's any additional information you can provide, we'd be grateful.

It might help if you could tell us the User IDs of affected users, which are strings like d904bd62-da08-416b-a816-ba797c9ee265 (it's safe to provide them; that one is mine.)  Do NOT provide "API Tokens".

The exact text of the bug reports might also give us clues.

**Comment by Alys** [comment]
I ran three Habitica-Beeminder goals for a few days each and every time I checked them they always had the correct information.

I'm closing this although if we get specific details we can reopen it.

Possible sources of confusion are that Beeminder goals do not count challenge To-Dos (because the number of those To-Dos is not fully under a player's control), and the goals can be set to count only To-Dos with a certain tag so if you forget which tag you chose (or accidentally chose the wrong one and didn't notice), then the data might seem incorrect.
